static/js/modules/nootils.js

```diff
@@ -6,8 +6,10 @@ module.exports = function(settings) {
   return {
     isTimely: function(date, event) {
       var due = new Date(date);
-      var start = this.addDate(null, event.start);
-      var end = this.addDate(null, (event.end + 1) * -1);
+      var start = new Date();
+      start.setUTCDate(start.getUTCDate() + event.start);
```

Comment:
That's what it was doing before, but I changed how addDate works so decoupled the two functions so the `isTimely` function would work the same as it used to.